app/src/main/java/org/greenstand/android/TreeTracker/api/models/requests/NewTreeRequest.kt

```diff
@@ -4,31 +4,31 @@ import com.google.gson.annotations.SerializedName
 
 data class NewTreeRequest(
     @SerializedName("user_id")
-    val userId: Int = 0,
+    val userId: Int,
     @SerializedName("uuid")
-    val uuid: String? = null,
+    val uuid: String,
     @SerializedName("lat")
-    val lat: Double = 0.toDouble(),
+    val lat: Double,
     @SerializedName("lon")
-    val lon: Double = 0.toDouble(),
+    val lon: Double,
     @SerializedName("gps_accuracy")
-    val gpsAccuracy: Int = 0,
+    val gpsAccuracy: Int,
```

Comment:
I believe gpsAccuracy is not relevant anymore given the gps convergence work, if I remember the conversation with @deepwinter on this correctly.  The references in the code  to gspAccuracy is still pending to be removed. Shouldn't we remove it from the NewTreeRequest too instead of making it a required field?